ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestContext.scala

```diff
@@ -11,13 +16,19 @@ private[testtool] final class LedgerTestContext private[infrastructure] (
     participants: Vector[ParticipantTestContext])(implicit ec: ExecutionContext) {
 
   private[this] val participantsRing = Iterator.continually(participants).flatten
-  private[this] def nextParticipant(): ParticipantTestContext =
-    participantsRing.synchronized { participantsRing.next() }
 
-  def participant(): Future[ParticipantTestContext] =
-    Future.successful(nextParticipant())
-
-  def participants(n: Int): Future[Vector[ParticipantTestContext]] =
-    Future.sequence(Vector.fill(n)(participant()))
+  def provision(allocation: ParticipantAllocation): Future[Participants] =
```

Comment:
Probably worth putting a comment here to explain what's going on.

Reply:
Done.